### PR TITLE
Fix server-side rendering

### LINKED ISSUE #6: Incompatibility with server-side rendering
Related:
https://github.com/yuanyan/boron/issues/8
https://github.com/yuanyan/boron/issues/24

### PR DESCRIPTION
**Pull request by LoicMahieu**
Fix #6

Related:
https://github.com/yuanyan/boron/issues/8
https://github.com/yuanyan/boron/issues/24

### CODE REVIEW

**Comment by baptwaels** [comment]
Hi,

Are you gonna release this modification to prevent server side rendering errors ?

It would really help on halogen !
thank you

**Comment by zvictor** [comment]
This one would be highly appreciated.

File: builtinStyle.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
 'use strict';
 
-module.exports = document.createElement('div').style;
+var canUseDOM = require('./canUseDOM');
+
+module.exports = canUseDOM && document.createElement('div').style;
```
Comment (zvictor) [inline builtinStyle.js:5]: I am having issues with this line of code as it sometimes evaluates to an object, sometimes to a `false`.
The following should be safer:

```js
module.exports = (canUseDOM && document.createElement('div').style) || {};
```

**Comment by hdngr** [comment]
For posterity - I merged this one here: https://github.com/hdngr/domkit

Thanks @yuanyan and @LoicMahieu!

**Comment by FrenchMajesty** [comment]
It would be nice to have this implemented. I'm still struggling with server pre-rendering despite a fix being written for it over a year ago.

**Comment by TaeKimJR** [comment]
@yuanyan Curious why this hasn't been merged in?

**Pull request merged**